content-repo/gendocs.py

```diff
@@ -578,7 +578,7 @@ def add_deprected_integrations_info(content_dir: str, deperecated_article: str,
         deperecated_info_file (str): json file with static deprecated info to merge
     """
     deprecated_infos = merge_deprecated_info(find_deprecated_integrations(content_dir), deperecated_info_file)
-    deprecated_infos = sorted(deprecated_infos, key=lambda d: d['name'].lower())  # sort by name
+    deprecated_infos = sorted(deprecated_infos, key=lambda d: d['name'].lower() if 'name' in d else d['id'].lower())  # sort by name
```

Comment:
@glicht I added this because the script was failing locally in my system as there were entries with just the `id` and not the `name`. Not sure how it worked in the original build.